rules/add-tracking-codes-in-urls/rule.md

```diff
@@ -20,11 +20,11 @@ Understanding  and taking  advantage of tracking codes on URLs is very important
 
 What is the difference in these 2 URLs?
 
-- **[https://www.thetrackr.com](https://www.thetrackr.com/)** 
+- **thetrackr\.com** 
 
 versus
 
-- **[https://www.thetrackr.com/bravo?utm\_expid=98763450-11.FrU2vkRZTECb\_wLuT78JGA.1&utm\_source=facebookads&utm\_medium=conversions&utm\_campaign=AU\_English\_BravoOrderSuccess4LAL\_AgeTargets&utm\_referrer=http%3A%2F%2Fm.facebook.com#ab\_test=b](https://www.thetrackr.com/bravo?utm_expid=98763450-11.FrU2vkRZTECb_wLuT78JGA.1&utm_source=facebookads&utm_medium=conversions&utm_campaign=AU_English_BravoOrderSuccess4LAL_AgeTargets&utm_referrer=http://m.facebook.com#ab_test=b)** 
+- **thetrackr\.com/bravo?utm\_expid=98763450-11.FrU2vkRZTECb\_wLuT78JGA.1&utm\_source=facebookads&utm\_medium=conversions&utm\_campaign=AU\_English\_BravoOrderSuccess4LAL\_AgeTargets&utm\_referrer=http%3A%2F%2Fm.facebook.com#ab\_test=b
```

Comment:
Add ending ** to make the url bold

Reply:
Done - added ending ** + replaced \. to &#46;